src/FNAPlatform/SDL2_FNAPlatform.cs

```diff
@@ -1409,23 +1409,26 @@ public static void SetRelativeMouseMode(bool enable)
 
 		private static string GetBaseDirectory()
 		{
-			if (	OSVersion.Equals("Windows") ||
-				OSVersion.Equals("Mac OS X") ||
-				OSVersion.Equals("Linux") ||
-				OSVersion.Equals("FreeBSD") ||
-				OSVersion.Equals("OpenBSD") ||
-				OSVersion.Equals("NetBSD")	)
-			{
-				/* This is mostly here for legacy compatibility.
-				 * For most platforms this should be the same as
-				 * SDL_GetBasePath, but some platforms (Apple's)
-				 * will have a separate Resources folder that is
-				 * the "base" directory for applications.
-				 *
-				 * TODO: Remove this and endure the breakage.
-				 * -flibit
-				 */
-				return AppDomain.CurrentDomain.BaseDirectory;
+			if (Environment.GetEnvironmentVariable("FNA_PREFER_SDL2_BASE_PATH") != "1")
+			{
+				if (	OSVersion.Equals("Windows") ||
+					OSVersion.Equals("Mac OS X") ||
+					OSVersion.Equals("Linux") ||
+					OSVersion.Equals("FreeBSD") ||
+					OSVersion.Equals("OpenBSD") ||
+					OSVersion.Equals("NetBSD")	)
+				{
+					/* This is mostly here for legacy compatibility.
```

Comment:
For some reason the editor didn't put the spaces back into the `*` lines... why are editors like this now?

Reply:
Probably my fault since I normally use 2 spaces for indentation so I had to change up Sublime to tabs and reindent it all. I added the spaces back in.